File: src/connect-k-java/Move.java
import java.util.*;

public class Move 
{
	public int row, col;

	public Move() 
	{
		this.col = 0;
		this.row = 0;
	}

	public Move(int col, int row) 
	{
		this.col = col;
		this.row = row;
	}

	public Move(String s)
	{
        Scanner reader = new Scanner(s);
        col = reader.nextInt();
		row = reader.nextInt();
		this.col = col;
		this.row = row;
	}

	public String toString()
	{
	    return col+" "+row;
	}
}

File: src/connect-k-cpp/Move.cpp
#include "Move.h"


Move::Move()
{
	row = 0;
	col = 0;
}
Move::Move(int col, int row){
	this->row=row;
	this->col=col;
}

Move::Move(std::string s)
{
    std::stringstream ss;
    int c,r;
    ss<<s;
    ss>>c>>r;
    this->row=r;
	this->col=c;

}
std::string Move::toString()
{
    std::string c,r;
    std::stringstream ss;
    ss<<col;
    c = ss.str();
    ss.str("");
    ss<<row;
    r = ss.str();
    return c+" "+r;
}



